user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_icons.md

```diff
@@ -0,0 +1,50 @@
+---
+uid: Working_with_trend_icons
+---
+
+# Working with trend icons
+
+> [!NOTE]
+>
+> - This feature requires [Storage as a Service](xref:STaaS) or a [self-hosted Cassandra-compatible database](xref:Supported_system_data_storage_architectures).
+> - You can enable or disable this feature via *System Center* > *System settings* > *analytics config*.
+
+The DataMiner Analytics software analyzes the behavior of numeric parameters in real time. A visual summary of the most recent behavior is shown in the form of a *trend icon* next to a trended parameter on a data page.
+
+Please note the following regarding these icons:
+
+- Behavior is calculated for single-value parameters and table parameters. For [partial tables](xref:Table_parameters#partial-tables), behavior is only calculated for the rows of the first page.
```

Comment:
The sentence about partial tables sounds off to me. We do show trend icons for every row that is viewed by the user? (which is indeed only the first page by default, but might be more or less depending on what the user actually looks at)